ydb/docs/en/core/maintenance/manual/backup-collections.md

```diff
@@ -0,0 +1,311 @@
+BACKUP `shop_backups`;
+```
+
+**Hourly incrementals with weekly full backups:**
+
+```sql
+-- Example: Sunday: Full backup (requires external scheduling)
+BACKUP `shop_backups`;
+
+-- Example: Monday-Saturday: Incremental backups (requires external scheduling)
+BACKUP `shop_backups` INCREMENTAL;
+```
+
+### Important scheduling notes {#important-scheduling-notes}
+
+- **External scheduling required**: YDB does not provide built-in scheduling. Use cron or similar tools.
+- **Background operations**: Backups run asynchronously and don't block database operations.
+- **Multiple collections**: Can run backups on different collections independently.
+
+## Monitoring backup operations {#monitoring}
+
+### Monitoring backup operations {#monitoring-backup-operations}
+
+```bash
+# Check backup operation status
+ydb operation list incbackup
+
+# Get details for specific operation
+ydb operation get <operation-id>
+```
+
+{% note warning %}
+
+Canceling backup operations is not yet supported. The `ydb operation cancel` command will return an error for backup operations.
+
+{% endnote %}
+
+```bash
+# Browse backup collections
+ydb scheme ls .backups/collections/
+
+# List backups in a collection
+ydb scheme ls .backups/collections/production_backups/
+
+# Get backup metadata
+ydb scheme describe .backups/collections/production_backups/backup_20240315_120000/
+```
+
+### Check operation status {#check-operation-status}
+
+All backup operations run in the background. Monitor their progress using:
+
+```bash
+# List all backup operations
+ydb operation list incbackup
+
+# Check specific operation status
+ydb operation get <operation-id>
+```
+
+### Browse backup structure {#browse-backup-structure}
+
+View backups through the database schema:
+
+```bash
+# List all collections
+ydb scheme ls .backups/collections/
+
+# View specific collection structure
+ydb scheme ls .backups/collections/shop_backups/
+
+# Check backup timestamps
+ydb scheme ls .backups/collections/shop_backups/ | sort
+```
+
+### Monitor backup chains {#monitor-backup-chains}
+
+Verify backup chain integrity:
+
+```bash
+# List backups in chronological order
+ydb scheme ls .backups/collections/shop_backups/ | sort
+
+# Check individual backup contents
+ydb scheme ls .backups/collections/shop_backups/20250821141519Z_full
+```
+
+## Managing backup collections {#managing-collections}
+
+### Collection information {#collection-information}
+
+Since collections are managed through SQL, browse them using schema commands:
+
+```bash
+# Browse all collections
+ydb scheme ls .backups/collections/
+
+# View collection directory contents
+ydb scheme describe .backups/collections/shop_backups/
+```
+
+### Collection status monitoring {#collection-status-monitoring}
+
+Check the health and status of your collections:
+
+1. **Verify recent backups exist**.
+2. **Check backup chain completeness**.
+3. **Monitor storage usage**.
+4. **Validate backup accessibility**.
```

Comment:
Provide specific instructions on how exactly to do that. Also, give recommendations that you should regularly check that the backups are restoring correctly with some sanity checks, again with examples/instructions.